content/articles/what-happens-when-a-domain-expires.markdown

```diff
@@ -0,0 +1,16 @@
+---
+title: What happens when a domain expires?
+excerpt: Description of the different statuses a domain can have after its expiration.
+categories:
+- Domains
+---
+
+# What happens when a domain expires?
+
+When a domain expires, it enters into a grace period. Most registry offer a non-guaranteed 29-day grace period for renewal for many TLDs. Each registry has different rules for their grace periods. There is no guaranteed grace period. 
```

Comment:
Some grammatical suggestions:

"When a domain expires, it may enter ~~into~~ a grace period. Most registr~~y~~ies offer a non-guaranteed 29-day grade period for renewal for many TLDs. There are no guaranteed grace periods."